docs/semgrep-supply-chain/triage-remediation.md

```diff
@@ -104,129 +91,49 @@ To assess your findings, Semgrep Supply Chain provides the following methods:
-   </td>
-  </tr>
-  <tr>
-   <td>Status
-   </td>
-   <td>Filters are based on the status of a vulnerability. The <strong>New</strong> filter is selected by default.
-   </td>
-  </tr>
-  <tr>
-   <td>Transitivity
-   </td>
-   <td>Filters are based on the transitivity of a vulnerability. <strong>All transitivities</strong> are selected by default.
-   </td>
-  </tr>
-  </tbody>
-</table>
-
-#### Exposure filters
-
-The following **exposure filters** are provided:
-
-
-<table>
-  <thead><tr>
-   <th>Exposure filter</th>
-   <th>Description</th>
-  </tr></thead>
-  <tbody><tr>
-   <td>Reachable
-   </td>
-   <td>Semgrep detected that there is a usage of the vulnerability from the dependency in your codebase. Some vulnerabilities are considered always reachable because they can be exploited regardless of their usage in your codebase. Others may require manual review and provide criteria to evaluate, such as whether the server running the vulnerable code is internet-accessible.
-   </td>
-  </tr>
-  <tr>
-   <td>Unreachable
-   </td>
-   <td>Semgrep determined that there is no usage of the vulnerability from the dependency in your codebase.
-   </td>
-  </tr>
-  <tr>
-   <td>Undetermined
-   </td>
-   <td>Semgrep does not scan for the reachability of this vulnerability.
-   </td>
-  </tr></tbody>
-</table>
-
-#### Status filters
+Use filters to narrow down your results. The following criteria are available for filtering:
 
-The following **status filters** are provided:
-
-<table>
-  <thead><tr>
-   <th>Status filter</th>
-   <th>Description</th>
-  </tr></thead>
-  <tbody><tr>
-   <td>New
-   </td>
-   <td>Vulnerabilities that have not undertaken triage or remediation action.
-   </td>
-  </tr>
-  <tr>
-   <td>Fixed
-   </td>
-   <td>Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as <strong>Fixed</strong>.
-   </td>
-  </tr>
-  <tr>
-   <td>Ignored
-   </td>
-   <td>Vulnerabilities that have been triaged as ignored by the user. Semgrep Supply Chain provides the following options for developers to select:
-    <ul>
-    <li>False positive</li>
-    <li>Acceptable risk</li>
-    <li>No time to fix</li>
-    </ul>
-   </td>
-  </tr></tbody>
-</table>
+| Filter                 | Description  |
+| ---------------------  | ------------ |
+| **Projects**           | Filter by repositories connected to your Semgrep account. |
+| **Branch**             | Filter by findings in different Git branches. |
+| **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with RBAC enabled. |
```

Comment:
```suggestion
| **Teams**              | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with [Teams](/deployment/teams#teams-beta) enabled. |
```
**Teams** is the term that is used in the app itself. AFAIK there's no mention of RBAC :)